### PR TITLE
upgrade jaeger version and add documentation for trace comparison

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Changes proposed in this pull request:

- upgrade jaeger version to 1.7.0
- add documentation for trace comparison

**Related issue(s)**
#1145

### CODE REVIEW

File: docs/tracing/docs/030-trace-comparison.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* In the comparison page, the first two traces (from the previous step) will be compared together. The first trace is called `A` and the second trace is called `B`.
+
+![Tracing architecture](assets/compare-traces-1.png)
+
+* There is an option to select which traces to compare from the top menus for `A` and `B`.
+
+![Tracing architecture](assets/compare-traces-2.png)
+
+* Trace spans have different colors which indicate different meaning:
+  * `Dark` colors indicate that the span is missing from one of the traces:
```
Comment (bszwarc) [inline docs/tracing/docs/030-trace-comparison.md]: `Dark` > Dark 
Generally we should remove the formatting for the color names.

File: docs/tracing/docs/030-trace-comparison.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    * `Dark red`: The span is only in trace `A`.
+    * `Dark green`: The span is only in trace `B`.
```
Comment (bszwarc) [inline docs/tracing/docs/030-trace-comparison.md]: `Dark green: The span is only present in trace **B**.`

File: docs/tracing/docs/030-trace-comparison.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  * `Light` colors indicate that the span is present in both traces but occurs more in one of the traces:
```
Comment (bszwarc) [inline docs/tracing/docs/030-trace-comparison.md]: `Light` > Light (no formatting)

File: docs/tracing/docs/030-trace-comparison.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    * `Light red`: The span in A has more spans than `B`.
```
Comment (bszwarc) [inline docs/tracing/docs/030-trace-comparison.md]: 1. `Light red: The span in **A** has more spans than **B**.`

2. Just to be sure: 
A span can have more additional spans?

File: docs/tracing/docs/030-trace-comparison.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    * `Light green`: The span in B has more spans than `A`.
```
Comment (bszwarc) [inline docs/tracing/docs/030-trace-comparison.md]: `Light green` > Light green (no formatting)

File: docs/tracing/docs/030-trace-comparison.md (hunk 4 above)
Comment (marcobebway) [reply docs/tracing/docs/030-trace-comparison.md]: Regarding point 2, the answer is yes, as per Jaeger blog post for this feature.

**Pull request merged**